### DISCUSSION TITLE
How to get the stable path refering for openblas library when building out of tree?

### DISCUSSION THREAD

**Discussion by hzhangxyz** [pyodide/pyodide#5051] [Q&A]
I need the openblas library file location when building my package, and previously, I use `.pyodide-xbuildenv/xbuildenv/pyodide-root/packages/.libs/lib/libopenblas.so`, it seems in the recent two versions, it has been changed to `.pyodide-xbuildenv-0.28.0/0.27.0a2/xbuildenv/pyodide-root/packages/.artifacts/lib/python3.12/site-packages/scipy.libs/libopenblasp-r0-01191904.3.27.so`. is there any canonical way to refer to this path?

**Comment by ryanking13** [comment]
No. We don't have a stable path for that. We are currently looking for a better way to deploy pre-built libraries so users can easily use it. But for now, I would recommend to build openblas yourselves.
